app/bundles/CoreBundle/IpLookup/IpapiLookup.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Mautic\CoreBundle\IpLookup;
+
+class IpapiLookup extends AbstractRemoteDataLookup
+{
+	/**
+	 * @return string
+	 */
+	public function getAttribution()
+	{
+		return '<a href="https://ipapi.com/" target="_blank">ipapi.com</a> real-time geolocation &
+		reverse IP Lookup REST API.';
+	}
+
+	/**
+	 * @return string
+	 */
+	protected function getUrl()
+	{
+		return 'http://api.ipapi.com/'.$this->ip.'?access_key='.$this->auth.'&output=json';
+	}
+
+	/**
+	 * @param $response
```

Comment:
This will keep the static analysis happy
```suggestion
	 * @param mixed[] $response
```